File: C/lista3 - vetores/exercicio_5.c
#include <conio.h>
#include <stdio.h>

int main() {
	int i, j, averageCountGreaterThan7 = 0;
	float grades[4], gradesAverage[10];
	
	for (i= 0; i< 11; i++) {
		for (j= 0; j< 4; j++) {
			printf("Informe a nota do aluno número %d: ", i);
			scanf("%f", &grades[j]);			
		}
		
		gradesAverage[i] = (grades[0] + grades[1] + grades[2] + grades[3]) / 4;		
	}
	
	for (i= 0; i< 11; i++) {
		if (gradesAverage[i] >= 7.0) {
			averageCountGreaterThan7++;
		}
	}
	
	printf("Numero de alunos com media maior que 7: %d", averageCountGreaterThan7);
	
	getch();
	return 0;
}

File: Python/lista3 - listas/exercicio_5.py
grades = []
for i in range(0, 10):
    student_grades = []
    for j in range(0, 4):
        student_grades.append(float(input('Informe a nota do aluno %.d: ' % (i + 1))))
    grades.append(student_grades)

average_students = 0
for student_grades in grades:
    sum_grades = 0
    for grade in student_grades:
        sum_grades += grade
    if (sum_grades / 4.0) >= 7.0:
        average_students += 1

print('%d alunos ficaram acima da media' % average_students)
